.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,211 @@
+          required: true
+  - type: input
+    id: affected-components
+    attributes:
+      label: Affected Component(s)
+      description: What is affected by this bug? Please describe it with only a few words here. Detailed description can be given later.
+      placeholder: No debug output is shown.
+    validations:
+      required: true
+  - type: textarea
+    id: when-does-it-occur
+    attributes:
+      label:  When does this occur and what happened?
+      description: Tell us what happened. Use [fenced code blocks](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks) when pasting lots of text!
+      placeholder: Although `DMS_DEBUG=1` is set, I see no debug output.
+      render: Markdown
+    validations:
+      required: true
+  - type: textarea
+    id: what-did-you-expect-to-happen
+    attributes:
+      label:  What did you expect to happen?
+      description: Tell us what you expected.
+      placeholder: I expected to see debug messages.
+      render: Markdown
+    validations:
+      required: true
+  - type: textarea
+    id: how-do-we-replicate-this-issue
+    attributes:
+      label: How do we replicate the issue?
+      description: What did you do and how can we replicate this issue?
+      value: |
+        1.
+        2.
+        3.
+        ...
+      render: Markdown
+    validations:
+      required: true
+  - type: input
+    id: mailserver-version
+    attributes:
+      label: DMS version
+      description: With which version did yor encounter this bug? Paste your image tag here.
+      placeholder: v10.1.2
+    validations:
+      required: true
+  - type: dropdown
+    id: main-memory
+    attributes:
+      label: How much RAM is available to DMS explicitly?
+      description: Please provide the amount of memory that DMS can really use. This includes main memory (RAM), but also swap or zram. If you're using Kubernetes, provide the resource limit here if you set one.
+      options:
+        - <= 1GB
+        - <= 2GB
+        - <= 4GB
+        - \> 4GB
+    validations:
+      required: true
+  - type: dropdown
+    id: cpu-count
+    attributes:
+      label: How many CPU cores / threads are available?
+      description: How many CPU cores or threads can DMS use? If you're using Kubernetes, provide the resource limit here if you set one.
+      options:
+        - <= 1 Core
+        - <= 2 Cores
+        - <= 4 Cores
+        - \> 4 Cores
+    validations:
+      required: true
+  - type: dropdown
+    id: system-type
+    attributes:
+      label: Is DMS running in a virtualized environment?
+      description: Choose the option that your setup is the most similar to. DMS is running on...
+      options:
+        - ... a physical host (with real, physical CPU cores)
```

aka Bare Metal? can dismiss, just suggesting due to the other two using title casing.

```suggestion
        - ... a Physical Host (with real, physical CPU cores)
```

---

I will instead make the other two lowercase 😆

---

Done. [998cb4b4884b4efcce5930e2c24d7cc902e658aa]